internal/future/statestore/redis.go

```diff
@@ -38,7 +40,7 @@ func (rb *redisBackend) Close() error {
 	return rb.redisPool.Close()
 }
 
-// NewRedis creates a storage.Service backed by Redis database.
+// NewRedis creates a statestore.Service backed by Redis database.
```

Comment:
This method is not exported. so lowercase the method name in the comment.

Reply:
Done.